FullEvents/AppDelegate.swift

```diff
@@ -7,58 +7,118 @@
 //
 
 import UIKit
+import Alamofire
+import SwiftyJSON
 import CoreData
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
+    
     var window: UIWindow?
-
-
+    
+    let notificationKey =  Notification.Name(rawValue: "Dismiss safari")
```

Comment:
Please use proper namings, it is not notification for dismiss safari. You should handle Login response.